statsmodels/stats/anova.py

```diff
@@ -293,12 +293,34 @@ def anova_lm(*args, **kwargs):
     Returns
     -------
     anova : DataFrame
-    A DataFrame containing.
+        When args is a single model, DataFrame contains.
+        sum_sq : float64
```

Comment:
should be float if scalar, not a numpy type.

Reply:
Thanks for the comment, and the initial work you did about a year ago.  Helped me get started on enhancing this docstring.

Unless I am understanding this incorrectly float64 will always return for every column.  I even tried a one item model based on the example in the docstring.

    >>> moore_lm = ols('conformity ~ fscore',  data=data).fit()
    >>> table = sm.stats.anova_lm(moore_lm,typ=2)
    >>> print(table)
                   sum_sq    df         F    PR(>F)
    fscore       0.134511   1.0  0.004784  0.945179
    Residual  1209.065489  43.0       NaN       NaN
    >>> table.dtypes
    sum_sq    float64
    df        float64
    F         float64
    PR(>F)    float64
    dtype: object

Reply:
I see, each is a vector.  So maybe `DataFrame with columns:` ?

Reply:
Added some clarity about the columns being returned.

commit https://github.com/statsmodels/statsmodels/pull/5233/commits/2e9cf7cc8d40931898dce3132b3c5324cb8e780a